pco_python/src/lib.rs

```diff
@@ -92,12 +166,21 @@ impl PyChunkConfig {
   /// :param delta_encoding_order: either a delta encoding level from 0-7 or
   /// None. If set to None, pcodec will try to infer the optimal delta encoding
   /// order.
-  /// :param int_mult_spec: either 'enabled' or 'disabled'. If enabled, pcodec
-  /// will consider using int mult mode, which can substantially improve
-  /// compression ratio but decrease speed in some cases for integer types.
-  /// :param float_mult_spec: either 'enabled' or 'disabled'. If enabled, pcodec
-  /// will consider using float mult mode, which can substantially improve
-  /// compression ratio but decrease speed in some cases for float types.
+  /// :param int_mult_spec: a IntMultSpec disabling, enabling, or providing a
+  /// base for modulo compression. If enabled, pcodec will consider using int
+  /// mult mode, which can substantially improve compression ratio but decrease
+  /// speed in some cases for integer types. If a base is provided, pcodec will
```

Comment:
Let's give some hint as to what cases these are. E.g. in the Rust docs we give a couple examples: https://docs.rs/pco/latest/pco/enum.IntMultSpec.html

Reply:
Will copy the explanations from the Rust docs.